### PR TITLE
Page not found following "Learn more about OpenShift update channels"

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/okd/issues/503

### CODE REVIEW

**Comment by mburke5678** [comment]
@vrutkovs PTAL. I removed references to the `fast-`, `candidate-`, and `eus-` channels in the update docs. 
The user can select these options in the web console, but no updates are available in OKD. Do we want to add an explicit message about this?  

Preview: http://file.rdu.redhat.com/~mburke/okd-issue-503/updating/updating-cluster-between-minor.html#update-upgrading-web_updating-cluster-between-minor

File: modules/understanding-upgrade-channels.adoc [hunk 1]
```diff
@@ -19,13 +19,23 @@ administrators explicitly decide to upgrade to the next minor version of
...
 * `candidate-{product-version}`
 * `fast-{product-version}`
 * `stable-{product-version}`
 * `eus-4.6` (only available when running 4.6)
 
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+{product-title} {product-version} offers the following upgrade channel:
+
+* `stable-{product-version}`
```
Comment (vrutkovs) [inline modules/understanding-upgrade-channels.adoc:34]: That would render as `stable-latest` on docs.okd.io, lets hardcode `stable-4`

File: modules/understanding-upgrade-channels.adoc [hunk 2]
```diff
@@ -71,18 +81,26 @@ never appear in the `stable-{product-version}` channel before they appear in the
...
 == stable-{product-version} channel
-
+ifndef::openshift-origin[]
 While the `fast-{product-version}` channel contains releases as soon
 as their errata are published, releases are added to the `stable-{product-version}` channel
 after a delay. During this delay, data is collected from Red Hat SRE teams, Red Hat support services,
 and pre-production and production environments that participate in connected customer program about the stability of the release.
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+Releases are added to the `stable-{product-version}` channel
```
Comment (vrutkovs) [inline modules/understanding-upgrade-channels.adoc:95]:
```suggestion
Releases are added to the `stable-4` channel
```

File: modules/understanding-upgrade-channels.adoc [hunk 3]
```diff
@@ -71,18 +81,26 @@ never appear in the `stable-{product-version}` channel before they appear in the
...
+after a delay. During this delay, data is collected from Red Hat SRE teams, Red Hat support services,
```
Comment (vrutkovs) [inline modules/understanding-upgrade-channels.adoc:96]:
```suggestion
after upgrade tests pass. 
```

File: modules/understanding-upgrade-channels.adoc [hunk 4]
```diff
@@ -71,18 +81,26 @@ never appear in the `stable-{product-version}` channel before they appear in the
...
+and pre-production and production environments that participate in connected customer program about the stability of the release.
```
Comment (vrutkovs) [inline modules/understanding-upgrade-channels.adoc:97]:
```suggestion

```
No data being collected for OKD by default

File: modules/understanding-upgrade-channels.adoc [hunk 5]
```diff
@@ -138,7 +164,22 @@ Red Hat will eventually provide supported update paths from any supported releas
 or `stable-{product-version}` channels to the latest release in {product-version}.z,
 although there can be delays while safe paths away from troubled releases are
 constructed and verified.
+endif::openshift-origin[]
+
+ifdef::openshift-origin[]
+The presence of an update recommendation in the `stable-{product-version}`
```
Comment (vrutkovs) [inline modules/understanding-upgrade-channels.adoc:170]:
```suggestion
The presence of an update recommendation in the `stable-4`
```

File: modules/understanding-upgrade-channels.adoc [hunk 6]
```diff
@@ -138,7 +164,22 @@ Red Hat will eventually provide supported update paths from any supported releas
...
+channel is a declaration that the update is fully supported while it is in the
+channel. While releases will never be removed from the channel, update recommendations
+that exhibit serious issues will be removed from the channel. Updates initiated
+after the update recommendation has been removed might not be supported.
+
+Red Hat will eventually provide supported update paths from any supported release in the
```
Comment (vrutkovs) [inline modules/understanding-upgrade-channels.adoc:176]:
```suggestion

```
OKD is not supported

File: modules/understanding-upgrade-channels.adoc [hunk 7]
```diff
@@ -155,6 +196,19 @@ and participating in Red Hat’s connected customer program. Red
 Hat uses this program to observe the impact of updates on your specific hardware and software
 configurations. Future releases might improve or alter the pace at which updates move
 from the `fast-{product-version}` to the `stable-{product-version}` channel.
+endif::openshift-origin[]
+
+ifdef::openshift-origin[]
```
Comment (vrutkovs) [inline modules/understanding-upgrade-channels.adoc:201]: Lets remove the whole section - we don't have a nice way to block the edges right now

**Comment by mburke5678** [comment]
manually cherrypicked to 4.6 via https://github.com/openshift/openshift-docs/pull/31694 and 4.5 via https://github.com/openshift/openshift-docs/pull/31690

**Pull request merged**